scripts/configure-common

```diff
@@ -1,3 +1,5 @@
+#!/bin/sh
```

Comment:
yes but in #1256 we agreed to add it because i think it doesn't hurt and supresses the checkbashisms warning.